Data/Vector/Generic.hs

```diff
@@ -1822,18 +1822,22 @@ scanr' f z = unstreamR . Bundle.scanl' (flip f) z . streamR
 -- | /O(n)/ Right-to-left scan over a vector with its index
 iscanr :: (Vector v a, Vector v b) => (Int -> a -> b -> b) -> b -> v a -> v b
 {-# INLINE iscanr #-}
-iscanr f z =
+iscanr f z v =
     unstreamR
```

Comment:
I think that's new with the bundle stuff. Unfortunately this function probably breaks fusion, because of the need for length information.